test-cypress/support/security-command.js

```diff
@@ -11,15 +11,56 @@ Cypress.Commands.add('switchOnSecurity', () => {
     });
 });
 
-Cypress.Commands.add('switchOffSecurity', () => {
+Cypress.Commands.add('loginAsAdmin', () => {
```

Comment:
This command is not implemented correctly. It must return the promise so that the chaining to work properly
```
     cy.loginAsAdmin().then(()=> {
                cy.deleteUser(user, true);
                cy.switchOffSecurity(true);
            });
```
Right now I'm not convinced that this would work reliably

Reply:
Same applies for the other new commands below!

Reply:
returns added